src/burst_buffer/burst_buffer.lua

```diff
@@ -688,7 +688,7 @@ function slurm_bb_job_teardown(job_id, job_script, hurry)
 	end
 
 	if ret == slurm.SUCCESS then
-		err = nil
+		err = ""
```

Comment:
To fix up the unit test, remove this line:
https://github.com/DataWorkflowServices/dws-slurm-bb-plugin/blob/main/testsuite/unit/src/burst_buffer/dws-test.lua#L840

You can then run this unit test with `make test-no-docker`.  That's a simple test, with no KIND, DWS, or slurm--everything is mocked.